src/views/v2/commerce/checkout/purchaser-info.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Purchaser Info.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/purchaser-info.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this [Global] Template object.
+ * @var bool $is_logged_out [Global] If the user is logged out.
+ * @var bool $must_login Global] Whether login is required to buy tickets or not.
+ */
+
+if ( ! $is_logged_out || $must_login ) {
+	return;
+}
+
+$title = __( 'Purchaser info', 'event-tickets' );
+?>
+
+<div class="tribe-tickets__form tribe-tickets__commerce-checkout-purchaser-info-field-wrapper">
+	<p class="tribe-common tribe-field-text"><?php echo esc_html( $title ); ?></p>
```

Comment:
This should be an `h4` as it comes after the checkout title, which is an h3.

Also, according to the designs, this should have the following classes: `tribe-common-h5`. We don't need to have `tribe-common` nor `tribe-field-text`. `tribe-common` works as a wrapper, and we already have that in upper elements. The other class doesn't belong here.